README.md

```diff
@@ -9,7 +9,7 @@
 PHP CDS is not a group or organisation, it's a concept. Its core idea is for the community to create RFCs,
 discuss them, then hold a vote on their acceptance as a community standard.
 
-Its **aim is to** be as loosely coupled to any individual, group or organisation as possible.
+Its **aim is** to be as loosely coupled to any individual, group or organisation as possible.
```

Comment:
No, `**aim is**` is correct, see the counterpart `**aim is not**` below. I just removed the word _to_ as it is not bold below.